redis-statsd.py

```diff
@@ -15,6 +16,8 @@
 
 args = parser.parse_args()
 
+logging.basicConfig(filename='/var/log/redis-statsd.log', filemode='a', format='%(levelname)s - %(message)s')
```

Comment:
nit: Should we pass the log file location as an argument?

Reply:
We would have to change how this python file is initialized which will another PR, I just want to fix this, afterwards I can change all the initialization to move it out